internal/test/e2e/e2e_test.go

```diff
@@ -363,9 +343,10 @@ func TestE2E(t *testing.T) {
 		t.Run("SampleCluster", func(t *testing.T) {
 			t.Parallel()
 			rl := corev1.ResourceList{
-				// 5 node cluster
-				corev1.ResourceCPU:    resource.MustParse("1000m"),
-				corev1.ResourceMemory: resource.MustParse("2000Mi"),
+				// 4 node cluster
+				// set job
+				corev1.ResourceCPU:    resource.MustParse("900m"),
+				corev1.ResourceMemory: resource.MustParse("850Mi"),
```

Comment:
Turns out that the circle-ci server where we run the e2e tests doesn't have enough CPU to run k8s + 5-node etcd cluster + in-cluster etcdctl pod. So I reduced the size of the etcd cluster here to 4. Hence the reduced resource quota.